src/app/shared/org-schema.service.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { inject, TestBed } from '@angular/core/testing';
-import { Organization } from './openapi';
+import { Organization } from './swagger';
```

Comment:
Is this swapping the "wrong" way on purpose?

Reply:
Yes, because I don't think it was actually working before.